checkov/common/output/report.py

```diff
@@ -307,6 +346,22 @@ def get_sarif_json(self, tool) -> Dict[str, Any]:
                     }
                 ],
             }
+
+            if record.check_result.get("result") == CheckResult.SKIPPED:
+                # sca_package suppressions can only be enabled via flag
```

Comment:
a SARIF report has a dedicated way on how to inform the user about suppression, it is added to the result under `suppressions` there you can add extra information like `justification`, which is our `suppress_comment` and a `kind`, which can have two different values `inSource` or `external`. Since we currently suppress/skip checks for IaC only via skip comment directly in the code, I default it to `inSource`. Using `--skip-check` for IaC results in not checking it at all, which is different for SCA and therefore it defaults to `external`